create-tokens.py

```diff
@@ -72,7 +74,10 @@ def create_tokens(ds, **kwargs):
         if dryRun is None:
             print('dryRun is None')
             return
-
+         if mintLimit is None:
```

Comment:
I don't know the busines logic behind this but I will recommend setting a default `minLimit` rather than exiting the job when `minLimit` is `None`.

Reply:
business logic works better with a limit every time. that is fine